lib/info.js

```diff
@@ -37,6 +37,7 @@ module.exports = function getInfo(link, options, callback) {
 
   var myrequest = options.request || request;
   var id = util.getVideoID(link);
+  var relatedIds = [];
```

Comment:
I don't agree with putting this here, it feels out of the way. Would make more sense if it came before the loop.